Makefile

```diff
@@ -91,10 +91,10 @@ welcome:
 install: node_modules
 
 # Simply running `make run` will spawn the Node.js server instance.
-run: welcome githooks install build
+run: welcome githooks install build server/devdocs/proptypes-index.json
```

Comment:
Superfluous due to https://github.com/Automattic/wp-calypso/pull/10342/files#diff-b67911656ef5d18c4ae36cb6741b7965R171 ?